From 998613543ec7da3f0198117f0545d0af26fb4cbb Mon Sep 17 00:00:00 2001
From: Varun <vyaparla@habana.ai>
Date: Tue, 28 Jun 2022 12:04:56 +0300
Subject: [PATCH] [SW-0] gaudi2: remove memories scrub todo's

This patch removed TODO comments of scrubbing VDEC, TPC and
XBAR memories. Scrubbing of XBAR memories are not required
since there is no partial write allowed to XBAR memories.
VDEC scrubbing is done as part of SW-88333. TPC unit reset
will init all of TPC memories.

Change-Id: I5bb67203aa442df7b5d29490aab1dd745223b51f
Signed-off-by: Varun <vyaparla@habana.ai>
---

diff --git a/arch/gaudi2/arc/asic.c b/arch/gaudi2/arc/asic.c
index cffe9e1..ceba629 100644
--- a/arch/gaudi2/arc/asic.c
+++ b/arch/gaudi2/arc/asic.c
@@ -1210,13 +1210,6 @@
 	gaudi2_mask_sram_ecc(false);
 }
 
-/* TODO: initialize internal tpc srams which
- * are not connected to reset block
- */
-
-/* TODO: Add Xbar memory init after bist to avoid ecc. */
-/* TODO: Add vdec memory init after bist to avoid ecc. */
-
 static void gaudi2_reset_engines(void)
 {
 	u32 tpc_iso_val, hbm_iso_val, nic_iso_val;
